drivers/dma/dma_silabs_siwx917.c

```diff
@@ -270,10 +271,11 @@ static int dma_channel_config(const struct device *dev, RSI_UDMA_HANDLE_T udma_h
 	} else {
 		channel_control.dstInc = UDMA_DST_INC_NONE;
 	}
-	status = UDMAx_ChannelConfigure(
-		&udma_resources, (uint8_t)channel, config->head_block->source_address,
-		config->head_block->dest_address, config->head_block->block_size, channel_control,
-		&channel_config, NULL, channel_info, udma_handle);
+	status = UDMAx_ChannelConfigure(&udma_resources, (uint8_t)channel,
+					config->head_block->source_address,
+					config->head_block->dest_address,
+					config->head_block->block_size, channel_control,
+					&channel_config, NULL, channel_info, udma_handle);
```

Comment:
This change revert the change in the previous commit. It make more sense to not modify it in the previous commit.